thispersondoesnotexist/online_getter.py

```diff
@@ -10,8 +10,8 @@ def get_online_person(params=None, **kwargs) -> bytes:
     :param kwargs: kwargs used by requests.get
     :return: the image as bytes
     """
-    r = requests.get("https://thispersondoesnotexist.com/", params, **kwargs)
-    return r.content
+    r = requests.get("https://thispersondoesnotexist.com/image", headers={'User-Agent': 'My User Agent 1.0'}).content
```

Comment:
Perfect, however I'd remove the headers part, if anybody wants to customize the User-Agent or anything, they can just pass it as a kwargs. Or we could use requests_html to have a randomized user agent.

Reply:
OH, I see! the problem is thispersondoesnotexist.com raise an error when no User-Agent is passed!